ETR01SDK-630: Docs: refactor CMake options mentions in tutorials (#537)
tropicsquare/libtropic: docs/tutorials/esp32/fw_update.md

    === ":fontawesome-brands-windows: Windows"
        TBA
    
!!! question "What if firmware update failed?"
    Check out the dedicated section in [FAQ](../../faq.md#fw-update-failed).

## Configuration
Besides the [Libtropic CMake options](../../reference/integrating_libtropic/how_to_configure/index.md) used to configure Libtropic, there are multiple CMake options specific to this example:

> - `LT_SH0_KEYS` (default: `"prod0"`) to choose which pairing keys in slot 0 will be used. Switch to engineering sample pairing keys if your TROPIC01 has them:

Reviewer: I ran this through a grammar check and this paragraph is a bit awkward. Maybe we can use sth like this:

```suggestion
- `LT_SH0_KEYS` (default: `"prod0"`) selects which pairing keys in slot 0 are used. Switch to engineering-sample pairing keys if your TROPIC01 is provisioned with them:
```

    ??? example "Switching to engineering sample pairing keys"
        === ":fontawesome-brands-linux: Linux"
            ```bash { .copy }
            idf.py -DLT_SH0_KEYS="eng_sample" build flash monitor
            ```

        === ":fontawesome-brands-apple: macOS"
            TBA

        === ":fontawesome-brands-windows: Windows"
            TBA
    Additionally, see [Default Pairing Keys for a Secure Channel Handshake](../../reference/default_pairing_keys.md) for more information.

> - `LT_DISABLE_MAINTENANCE_MODE` (default: `"ON"`) to control whether the FW update example disables Maintenance Mode in R-Config after a successful FW update. Set to `"OFF"` or `0` if your TROPIC01 has Maintenance Mode enabled and you don't want the FW update example to disable it:

Reviewer: ```suggestion
- `LT_DISABLE_MAINTENANCE_MODE` (default: `"ON"`) controls whether the FW update example disables Maintenance Mode in R-Config after a successful FW update. Set to `"OFF"` or `0` if Maintenance Mode is enabled on your TROPIC01 and you don't want the FW update example to disable it:
```
